scripts/scale.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+export VALID_NAMESPACES=("hocs-gamma" "hocs-delta" "hocs-qax")
+
+current=$(kubectl config current-context)
```

Comment:
This won't work in-pod, just on a dev machine

Reply:
Yes I am aware of that. We need to keep the script somewhere, and this seems to be the most logical place.
Users can copy it from here to their local machines.